src/bonefish/trace/trace.hpp

```diff
@@ -54,14 +69,38 @@ inline const char* base_file_name(const char* file_path)
 
 #define BONEFISH_TRACE_NOARGS(fmt) \
     if (bonefish::trace::is_enabled()) { \
-        std::cerr << "[" << bonefish::trace::base_file_name(__FILE__) << ":" << __LINE__ << "][" << __FUNCTION__ << "] " \
-                << boost::format(fmt) << std::endl; \
+        std::ostringstream oss; \
+        oss << "[" << bonefish::trace::base_file_name(__FILE__) << ":" << __LINE__ << "][" << __FUNCTION__ << "] " \
+                << boost::format(fmt); \
```

Comment:
nit: IMHO: better to have some standard width prefix to avoid searching where logs started,

Timestamp process_name [pid](TID) : log_level : MSG FILE_NAME LINE